### DISCUSSION TITLE
Please bring back the Filter Channel Input box when Entries is hovered at the sidebar.

### DISCUSSION THREAD

**Discussion by chunkitdesign** [ExpressionEngine/ExpressionEngine#2903] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
In EE5, when the label of Edit is hovered, there is a Filter Channel Input box at top; and this is very useful when we have more than 70 channels. However, since upgrading to EE6, this feature is gone. Right now, all 70+ channels are shown when the Entries are hovered. It is very overwhelming to my clients. May you please bring back this feature and also limit the height of the List Container.

**Describe the solution you'd like**
n/a

**Describe alternatives you've considered**
Consider to bring this back.

[EE5 Entries List Screenshot](https://www.dropbox.com/s/zx73m2i7stpgzye/channel_list-wFilter_ee5.pg.png?dl=0)
[EE6 Entries List Screenshot](https://www.dropbox.com/s/fc8xyl00n3t90lo/channellist_long_ee6.png?dl=0)

**Comment by intoeetive** [comment]
@chunkitdesign while not direct solution, just mentioning that editors can still go to "all entries" page and filter the channels there

**Comment by chunkitdesign** [comment]
@intoeetive, thanks for your reply. The issue is that since we have more than 73 channels, hovering over "Entries" will create a long list which overwhelms the editors. In EE5, the listing box is limited to a certain height with a Filter input box which is very useful. Instead of scrolling through the long list to find the channel entry one wants to add or edit, I type "TH," and all the theatre-related channels would show up. Neat and convenient! I Hope that the EE team will consider bringing this UI back. Thanks!

**Comment by intoeetive** [comment]
Not really solving the problem of searchability, but in https://github.com/ExpressionEngine/ExpressionEngine/pull/3940 I'm adding ability to sort the channels, which will make it possible to put those that are used the most at the top, instead of alphabetical sorting
